src/Raven.Server/Documents/Handlers/Admin/RachisAdminHandler.cs

```diff
@@ -335,6 +336,16 @@ public async Task AddNode()
             {
                 requestExecutor.DefaultTimeout = ServerStore.Engine.OperationTimeout;
 
+                // test connection to remote.
+                var result = await ServerStore.TestConnectionToRemote(nodeUrl, database: null);
+                if (result.Success)
+                {
+                    // test connection from remote to destination
+                    result = await ServerStore.TestConnectionFromRemote(requestExecutor, ctx, nodeUrl);
+                    if(result.Success == false)
+                        throw new InvalidOperationException(result.Error);
```

Comment:
Here need to indicate that we were able to connect to the remote, and it wasn't able to connect to us

Reply:
The `result.Error` already contains this info.